### PR TITLE
JMediawiki package

### PR DESCRIPTION
**Pull request by nprasath002**
JMediawiki package with unit tests

### CODE REVIEW

File: libraries/joomla/mediawiki/mediawiki.php [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+ * @subpackage  MediaWiki
+ *
+ * @copyright   Copyright (C) 2005 - 2012 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE
+ */
+
+defined('JPATH_PLATFORM') or die;
+
+/**
+ * Joomla Platform class for interacting with a GitHub server instance.
```
Comment (elinw) [inline libraries/joomla/mediawiki/mediawiki.php]: Here and a few other places you have  GitHub not mediawiki

**Comment by nprasath002** [comment]
corrected all typos. thanks

**Comment by pasamio** [comment]
Looks like the unit tests have errors:
http://developer.joomla.org/pulls/pulls/1547.html

**Comment by LouisLandry** [comment]
@nprasath002 I'd love to get this ready for merging.  Can you get the unit test and code style issues sorted out for us so we can give it a proper once over?

_Note: I'm closing this pull request for now until the requested changes are uploaded. Please re-open the pull request at that time so we can review it again. Thanks._

**Pull request closed without merging**